### PR TITLE
[IMP] emc: get journal by xml id rather than code

### PR DESCRIPTION
**Pull request by robinkeunen**
small pr for a small improvement.

L'avantage: if dummy dude changes the journal code, it won't fail

### CODE REVIEW

**Comment by houssine78** [comment]
nice catch ! I've done it for all the email template. forgot to do it for journal

**Review by houssine78 — APPROVED** [review]

File: easy_my_coop/demo/coop.xml [hunk 1]
```diff
@@ -70,7 +70,7 @@
     </record>
 
     <record id="account_cooperator_demo" model="account.account">
-        <field name="code">416101</field>
+        <field name="code">416000</field>
         <field name="name">Cooperators</field>
         <field name="user_type_id" ref="account.data_account_type_receivable"/>
         <field name="reconcile" eval="True"/>
```
Comment (houssine78) [inline easy_my_coop/demo/coop.xml]: maybe that this should go in easy_my_coop_l10n_be...

Comment (robinkeunen) [reply easy_my_coop/demo/coop.xml]: But I need it to initiate a demo db, I don't want to depend on l10n_be. What could be a good value for demo data?

**Comment by houssine78** [comment]
the account 416 is for belgian accounting chart. so if you need to test this, the corresponding test should go in localization. And I would says that one of the localization is mandatory. Installing easy_my_coop should be by installing localization be, ch or fr.

Anyway this something that could be done later.

**Review by mclaeysb — APPROVED** [review]

**Pull request merged**